llvm/include/llvm/ADT/MapVector.h

```diff
@@ -99,6 +99,11 @@ class MapVector {
     return try_emplace_impl(Key).first->second;
   }
 
+  [[nodiscard]] inline auto keys() { return make_first_range(Vector); }
+  [[nodiscard]] inline auto values() { return make_second_range(Vector); }
+  [[nodiscard]] inline auto keys() const { return make_first_range(Vector); }
+  [[nodiscard]] inline auto values() const { return make_second_range(Vector); }
```

Comment:
```suggestion
  [[nodiscard]] inline auto keys() { return make_first_range(Vector); }
  [[nodiscard]] inline auto keys() const { return make_first_range(Vector); }
  [[nodiscard]] inline auto values() { return make_second_range(Vector); }
  [[nodiscard]] inline auto values() const { return make_second_range(Vector); }
```

Keep values(), keys() together?